load.php

```diff
@@ -5,13 +5,15 @@
 use HM\Platform;
 
 require_once __DIR__ . '/inc/namespace.php';
+require_once __DIR__ . '/inc/passwords/namespace.php';
 
 add_action( 'hm-platform.modules.init', function () {
 	$default_settings = [
-		'enabled'                 => true,
-		'require-login'           => ! in_array( Platform\get_environment_type(), [ 'production', 'local' ], true ),
-		'audit-log'               => true,
-		'2-factor-authentication' => true,
+		'enabled'                   => true,
+		'require-login'             => ! in_array( Platform\get_environment_type(), [ 'production', 'local' ], true ),
+		'audit-log'                 => true,
+		'2-factor-authentication'   => true,
+		'minimum-password-strength' => 2,
```

Comment:
Above you say `@param int $minimum_strength Minimum required strength. Default is 3.`, but this doesn't appear to be the case?